spec/rubocop/cop/rspec/example_length_spec.rb

```diff
@@ -56,6 +56,36 @@
     end
   end
 
+  context 'given skipped nested examples' do
+    it 'does not flag the example' do
+      expect_no_offenses(<<-RUBY)
+        skip do
+          line 1
+          it do
+            line 1
+            line 2
+            line 3
+          end
+        end
+      RUBY
+    end
+  end
+
+  context 'given pending nested examples' do
+    it 'does not flag the example' do
+      expect_no_offenses(<<-RUBY)
+        pending do
+          line 1
+          it do
+            line 1
+            line 2
+            line 3
+          end
+        end
```

Comment:
If you remove `line 1`:
```
     # No reason given
     Failure/Error:
       it do
         line 1
         line 2
         line 3
       end

       `it` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
```

`pending`/`skip` are aliases of `example`, not `example_group`.
And they are not supposed to have nested examples.

Reply:
Thanks @pirj  for your answer. 
I was not aware about that, feel like RSpec is really permissive with the usage of `pending`/`skip` with `example_group`,  and can easily lead to this misunderstanding. 

> And they are not supposed to have nested examples.

Seems like it's a good flag for a missing cop, does it make sense to add cop to flag an `example_group` using these methods?